Assets/Scripts/VrSdk.cs

```diff
@@ -184,19 +184,35 @@ void Awake()
             if (App.Config.m_SdkMode == SdkMode.Oculus)
             {
 #if OCULUS_SUPPORTED
-      // ---------------------------------------------------------------------------------------- //
-      // OculusVR
-      // ---------------------------------------------------------------------------------------- //
-      OVRManager manager = gameObject.AddComponent<OVRManager>();
-      manager.trackingOriginType = OVRManager.TrackingOrigin.FloorLevel;
-      manager.useRecommendedMSAALevel = false;
-
-      SetControllerStyle(TiltBrush.ControllerStyle.OculusTouch);
-      // adding components to the VR Camera needed for fading view and getting controller poses.
-      m_VrCamera.gameObject.AddComponent<OculusCameraFade>();
-      m_VrCamera.gameObject.AddComponent<OculusPreCullHook>();
-
-      gameObject.AddComponent<OculusMRCCameraUpdate>();
+                // ---------------------------------------------------------------------------------------- //
```

Comment:
The reformatting here was done manually, and wasn't caught in #81 because of https://github.com/dotnet/format/issues/1101 (and probably the same basic issue in Rider)